packages/devtools/lib/src/memory/memory_plotly.dart

```diff
@@ -409,6 +407,16 @@ class MemoryPlotly {
 class EventTimeline {
   EventTimeline(this._domName, this._chart);
 
+  // Light blue and Dark Blue 600 dark mode colors see
```

Comment:
nit: replace with "Light theme is Light Blue and dark theme is Dark Blue 600. See https:..."

Reply:
Cleanup this and the other comments below to match.